docs/source/releases.rst

```diff
@@ -1,6 +1,68 @@
-=================
- Release History
-=================
```

Comment:
Title was adjusted to use the pcdsdevices style, as the pre-release document aggregation script butchered the title when I ran it.